deno_dist/helper/ssg/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Buffer } from "node:buffer";
+import * as path from 'node:path'
+import { inspectRoutes } from '../../helper/dev/index.ts'
+import type { Hono } from '../../hono.ts'
+import type { Env, Schema } from '../../types.ts'
+
+export interface FileSystemModule {
+  writeFile(path: string, data: string | Buffer): Promise<void>
+  mkdir(path: string, options: { recursive: boolean }): Promise<void | string>
+}
+
+export interface ToSsgResult {
```

Comment:
If you made `ToSsg` into `ToSSG`, you might as well make `ToSsgResult` into `ToSSGResult` too. Like `URL` and `URLSearchParams`.